src/ui/components/Initiatives/InlineKeyField.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import React from "react";
+import {v4 as uuid} from "uuid";
+import TextField from "@material-ui/core/TextField";
+import {useInput} from "react-admin";
+
+type Props = {|
+  source: string,
+  label: string,
+|};
+
+export const InlineKeyField = (props: Props) => {
+  const key = React.useMemo(() => uuid(), []);
+
+  const {
+    id,
+    input,
+    meta: {touched, error},
+    isRequired,
+  } = useInput({
+    ...props,
+    initialValue: key,
+  });
+
+  return (
+    <TextField
```

Comment:
Is this something we want to hide? I hid it because I figured users wouldn't want to see uuid fields, but I'm open to the idea of leaving it as shown, especially now that this bug was caught.

Reply:
Yea, lets leave it shown for now since its still going to be used by devs mostly, can hide it later once we are confident things are working properly. Will be handy for making sure things are wired up correctly if we can see the IDs in the UI